modules/ols-creating-the-credentials-secret-using-cli.adoc

```diff
@@ -34,10 +32,21 @@ stringData:
 ----
 <1> The `apitoken` is not `base64` encoded.
 +
-[NOTE]
-====
-The YAML element is always `apitoken` regardless of what the LLM provider calls the access details.
-====
+.Example credential secret for {azure-official} {openai}
++
+[source,yaml, subs="attributes,verbatim"]
+----
+apiVersion: v1
+data:
+  client_id: <base64 encoded client id>
```

Comment:
```suggestion
  client_id: <base64_encoded_client_id>
```
These variables are all missing underscores

Reply:
Thanks for noticing this -- I made the update here as well as on the task using the web console.